include/executors/kernel_constructor.h

```diff
@@ -33,10 +33,17 @@ namespace blas {
 /*!using_local_memory.
  * @brief Indicates whether if the kernel uses shared memory or not.
  This is a work-around for the following enum class.
+ //enum class using_shared_mem { enabled, disabled };
```

Comment:
This isn't needed, right?

Reply:
I believe it's the `enum class` referred to by the rest of the comment

Reply:
If the member of the `enum` class changed can you either remove this or update the comment

Reply:
@CodeplayBen do you know more about this, was this what is now named  `using_local_memory`?

Reply:
Seems like it but it's been this way for 2 years at least according to the history. The enum class is mentioned in a few other comments in this file so I think they've just not been updated/tidied up correctly. I would probably remove this line specifically as I don't think it's necessary.

Reply:
Ok, line removed